bciers/apps/administration/tests/components/facilities/FacilitiesDataGrid.test.tsx

```diff
@@ -175,4 +176,52 @@ describe("FacilitiesDataGrid component", () => {
       ).toBe("facility 1");
     });
   });
+
+  it("opens Action Cell links in new tab when navigating from Registration", async () => {
+    render(
+      <FacilityDataGrid
+        operationId="randomOperationUUID"
+        initialData={mockResponse}
+        fromRegistration={true}
+      />,
+    );
+
+    const actionCells = await screen.findAllByText("View Details");
+    expect(actionCells.length).toBe(mockResponse.rows.length);
+
+    actionCells.forEach(async (actionCell) => {
+      expect(actionCell).toHaveAttribute("target", "_blank");
+      expect(actionCell).toHaveAttribute("rel", "noopener noreferrer");
+
+      // check for the tooltip text
+      userEvent.hover(actionCell);
+      expect(
+        await screen.findByText(/Link opens in a new tab/i),
+      ).toBeInTheDocument();
```

Comment:
This may be passing because it's `toBeInTheDocument`.  `toBeVisible` might more accurately simulate a user